docs/exploration-and-organization/collections.md

```diff
@@ -72,7 +72,7 @@ You can upload CSV data by clicking on the **Upload** icon in the top right of t
 
 Metabase will create a [model](../data-modeling/models.md) that contains that CSV data, as well as the model's underlying table.
 
-Uploads will only be available if your admin has enabled uploads for your Metabase. See [Uploading data](../databases/uploads.md).
+Uploads will only be available if your admin has enabled uploads for your Metabase, and you're in a group with unrestricted access to the database used to store those uploads. See [Uploading data](../databases/uploads.md).
```

Comment:
We are still figuring out this ([context](https://metaboat.slack.com/archives/C04S696LRUM/p1688502771309619))

I think we will converge to require `unrestricted` access to the upload schema.